src/test/java/org/zalando/nakadi/util/HashGeneratorTest.java

```diff
@@ -0,0 +1,65 @@
+package org.zalando.nakadi.util;
+
+import com.google.common.collect.ImmutableList;
+import org.junit.Test;
+import org.zalando.nakadi.domain.SubscriptionBase;
+import org.zalando.nakadi.view.SubscriptionCursorWithoutToken;
+
+import static com.google.common.collect.Sets.newHashSet;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class HashGeneratorTest {
```

Comment:
I missed the output of the method `generateSubscriptionKeyFieldsHash` being written at least once in tests.